src/rstudioapi.ts

```diff
@@ -417,7 +416,7 @@ async function reuseOrCreateEditor(targetDocument: TextDocument) {
 
 
   const matchingTextEditors = KnownEditors.filter((editor) =>
-      editor.document.uri.toString === targetDocument.uri.toString);
+    editor.document.uri.toString === targetDocument.uri.toString);
```

Comment:
Should be `editor.document.uri === targetDocument.uri`? or both `toString()`.

Reply:
Oh wow. That's incredible.That was just a style change, but what a find.

Reply:
I think toString works best as `uri` is listish data structure.